packages/api/src/router/user.ts

```diff
@@ -130,6 +134,125 @@ export const userRouter = createTRPCRouter({
       }
     }),
 
+  payCoinsByUserId: publicProcedure
+    .input(
+      z.object({
+        receiver: z.object({
+          id: z.string(),
+          username: z.string(),
+          avatar: z.nullable(z.string()),
+          discriminator: z.string(),
+        }),
+        sender: z.object({
+          id: z.string(),
+          username: z.string(),
+          avatar: z.nullable(z.string()),
+          discriminator: z.string(),
+        }),
+        coins: z.number(),
+      }),
+    )
+    .mutation(async ({ ctx, input }) => {
+      try {
+        /**
+         * This function set a temp thumbnail for the user
+         */
+        const tempThumbnail = setTempThumbnail(input.receiver);
+
+        /**
+         * Check in the user, have a balance for the transaction
+         */
+        const sender = await ctx.prisma.user.findUnique({
+          where: { discordId: input.sender.id },
+          select: {
+            coins: true,
+          },
+        });
+
+        /**
+         * If sender not exist, create a new user and return error tokens message
+         */
```

Comment:
```suggestion
        // If sender not exist, create a new user and return error tokens message
```

Por que devolveria un error al crear el nuevo usuario, no entendi esa parte?